packages/aws-cdk-lib/aws-cloudfront/test/distribution.test.ts

```diff
@@ -469,14 +469,14 @@ describe('certificates', () => {
       DistributionConfig: {
         Aliases: Match.absent(),
         ViewerCertificate: {
-          AcmCertificateArn: 'arn:aws:acm:us-east-1:123456789012:certificate/12345678-1234-1234-1234-123456789012',
+          AcmCertificateArn: 'arn:aws:acm:us-east-1:123456789012:certificate/12345678-123456789012-123456789012-123456789012-123456789012',
```

Comment:
The `1234` here was not an account ID, so I'm not sure why it is file is changed.

Did you do a mass search-replace over the entire codebase?